### PR TITLE
Add access to more talons

### PR DESCRIPTION
**Pull request by jpmii**
## Description

Add access to Checkout useFlow, Checkout useOverview and CartPage useProduct talons

## Related Issue

#1784

## Acceptance

@jimbo @supernova-at @tjwiebell

### CODE REVIEW

File: packages/peregrine/lib/targets/TalonWrapperConfig.js [hunk 1]
```diff
@@ -33,6 +33,22 @@ class TalonWrapperConfig {
...
+         * @instance
+         */
+        this.Checkout = {
+            /**
+             * useFlow {@link WrappableTalon} object for the `useFlow()` talon
+             * useOverview {@link WrappableTalon} object for the `useOverview()` talon
+             * @type {WrappableTalon}
+             */
+            useFlow: new WrappableTalon(addTransforms, 'Checkout/useFlow'),
+            useOverview: new WrappableTalon(
```
Comment (sirugh) [inline packages/peregrine/lib/targets/TalonWrapperConfig.js:47]: These are legacy components - do you still need them targetable?

File: packages/peregrine/lib/targets/TalonWrapperConfig.js [hunk 2]
```diff
@@ -61,6 +77,23 @@ class TalonWrapperConfig {
                 'MiniCart/useMiniCart'
             )
         };
+        /**
+
+         * Provides access to the talon used in Venia's CartPage UI component.
```
Comment (sirugh) [inline packages/peregrine/lib/targets/TalonWrapperConfig.js:82]:
```suggestion
         * Provides access to the talons used in Venia's CartPage UI component.
```

File: packages/peregrine/lib/targets/TalonWrapperConfig.js [hunk 3]
```diff
@@ -61,6 +77,23 @@ class TalonWrapperConfig {
...
+         * @instance
+         */
+        this.CartPage = {
+            /**
+             * useProduct {@link WrappableTalon} object for the `useProduct()` talon
```
Comment (sirugh) [inline packages/peregrine/lib/targets/TalonWrapperConfig.js:87]: This jsdoc belongs one layer lower, below `ProductListing: {` I think?

**Comment by jimbo** [comment]
After merging #2765 , all talons can now be tapped, so we shouldn't need to add them explicitly anymore.

```es6
const peregrineTargets = targets.of('@magento/peregrine')
const talonsTarget = peregrineTargets.talons

talonsTarget.tap(talons => {
  talons.App.useApp.wrapWith('@my-extensions/log-wrapper')
})
```

**Pull request closed without merging**